pkg/cmd/auth/login/login.go

```diff
@@ -239,19 +259,21 @@ func loginRun(opts *LoginOptions) error {
 		}
 	}
 
-	var gitProtocol string
-	err = prompt.SurveyAskOne(&survey.Select{
-		Message: "Choose default git protocol",
-		Options: []string{
-			"HTTPS",
-			"SSH",
-		},
-	}, &gitProtocol)
-	if err != nil {
-		return fmt.Errorf("could not prompt: %w", err)
-	}
+	gitProtocol := "https"
```

Comment:
I'm thinking that we should skip the `gh config set git_protocol` step entirely in non-interactive mode. Setting `git_protocol` to HTTPS by default has no benefit since that's already the default, and as @tierninho points out, if someone has had SSH set up for this host, this logic will reset it.